OxfordBodleian/Juel-Jensen/BDLjj29.xml

```diff
@@ -75,44 +91,71 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="left">30</dim>
                                         <dim type="intercolumn">20</dim>
                                     </dimensions>
+                                    <note>The characters per column are 7-9.</note>
+                                    <ab type="pricking">Ruling is visible; pricking is visible.</ab>
+                                    <ab type="ruling" subtype="pattern">  
+                                        <locus from="1ra" to="1vb"/>    
+                                        Ruling pattern: 1A-1A-1A1A/0-0/0-0/C.</ab>
+                                    <ab type="pricking">Primary pricks are visible.</ab>
+                                    <ab type="pricking">Ruling pricks are visible.</ab>
+                                      <!--<ab type="ruling">The upper line is written below the ruling.</ab>--> <!-- please check, I do not see it well-->
+                                    <ab type="ruling">The bottom line is written above the
+                                        ruling.</ab>
                                 </layout>
                             </layoutDesc>
-                           
+                            
                         </objectDesc>
                         
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+                                <date notBefore="1650" notAfter="1775"/>
+                                <seg type="script"><foreign xml:lang="gez">Gʷǝlḥ</foreign>-script.</seg>
+                                <desc>Fine and regular hand, broad spaces between the characters.</desc>
+                                <seg type="ink">Black, red.</seg>
+                                <seg type="rubrication">Elements of punctuation signs, a numeral, holy name (<persName ref="PRS6819Mary"/>).</seg>
```

Comment:
So, do we provide the individual names in red ink?
I read in the meeting agenda: Holy names as a general term, not listing the individual names

Reply:
Yes, but since here there is exactly one name I found it read strange so generic (just "holy name"?)...

Reply:
What about `the name of <persName ref="PRS6819Mary"/>`?
That is, in one-name occurrences we might establish to write that name explicitly and leave out "holy name" as well.

Reply:
I wrote in another ms-entry. "Rubricated are ... ; the name of `<persName ref="PRS6819Mary"/>`